docs/source/builtin-plugins.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Apollo Server plugins
+sidebar_title: Overview
+---
+
+**Plugins** extend Apollo Server's core functionality by performing custom operations in response to certain events. These events correspond to individual phases of the GraphQL request lifecycle, and to the lifecycle of Apollo Server itself.
+
+Certain Apollo Server features are distributed as plugins in the `apollo-server` library, some of which are installed automatically in certain cases. You can also [create custom plugins](./integrations/plugins/).
```

Comment:
it's more like, some features are distributed as plugins in the `apollo-server-core` (need `-core`!) package and are installed automatically in certain cases. Others are distributed in their own packages (such as `apollo-server-plugin-response-cache`)

Reply:
Also should probably note that the logic for when these are installed is described at the top of each doc section. Or maybe it would be interesting to add that to the table in the next section too?